output/v0.0.json

```diff
@@ -3376,6 +3376,21 @@
                         "readOnly": false,
                         "maxLength": 250,
                         "description": "This sensitive information is stored encrypted and can only be retrieved through the API."
+                    },
+                    "payment_group_id": {
+                        "type": "string",
+                        "description": "",
+                        "maxLength": 100
+                    },
+                    "can_be_resubmitted": {
+                        "type": "boolean",
+                        "description": ""
```

Comment:
Description: Indicates if the same invoice number be submitted again. Resubmitting the same invoice number for a specific buyer and supplier is only allowed if the prior payment was refunded.